server/src/main/java/org/opensearch/identity/package-info.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/** Identity principal class for owners of request to/from extensions */
```

Comment:
Package info shouldn't reference a specific class, but the type of classes in this package.